colossalai/fx/tracer/meta_patch/patched_bias_addition_module/conv.py

```diff
@@ -0,0 +1,53 @@
+import torch
+import torch.nn.functional as F
+
+from ..registry import bias_addition_module
+from .bias_addition_module import BiasAdditionModule
+
+
+class BiasAdditionConv(BiasAdditionModule):
+
+    def create_non_bias_func_proxy(self):
+        return super().create_non_bias_func_proxy()
+
+    def create_bias_addition_proxy(self, non_bias_func_proxy, bias_proxy):
+        return super().create_bias_addition_proxy(non_bias_func_proxy, bias_proxy)
```

Comment:
If these two methods are standard, there is no need to annotate them with `abstract` method.